Example/Segment-Nielsen-DCR/SEGNielsenDCRAppDelegate.m

```diff
@@ -7,13 +7,15 @@
 //
 
 #import "SEGNielsenDCRAppDelegate.h"
-#import "SEGNielsenDCRIntegrationFactory.h"
+
+@import NielsenAppApi;
+@import Segment_Nielsen_DCR;
```

Comment:
Is this an umbrella header? Curious why naming changed?

Reply:
No, it's just a way to import a framework without having to get into header specifics.  Due to our naming, -'s are converted to _ and so this just makes it easier because we really don't care, we just need access to stuff in the framework.